### PR TITLE
On Win32, write out default .hyper.js as CRLF so Notepad doesn't have a fit

### PR DESCRIPTION
**Pull request by paulcbetts**
Shouldn't have any adverse effects and will make Windows users' first experience a bit nicer

### CODE REVIEW

File: app/config.js [hunk 1]
```diff
@@ -84,6 +84,10 @@ function exec(str) {
   return true;
 }
 
+function crlfify(str) {
```
Comment (rauchg) [inline app/config.js]: can we add a comment on what this does and why it's here?

Comment (paulcbetts) [inline app/config.js]: :+1:

**Comment by matheuss** [comment]
Friendly ping @paulcbetts 🙌

File: app/config.js (hunk 1 above)
Comment (rauchg) [inline app/config.js]: @paulcbetts 👍

**Comment by matheuss** [comment]
Thank you @paulcbetts 🙆

**Pull request merged**